cg/meta/demultiplex/utils.py

```diff
@@ -95,3 +95,8 @@ def parse_flow_cell_directory_data(
 ) -> FlowCellDirectoryData:
     """Return flow cell data from the flow cell directory."""
     return FlowCellDirectoryData(flow_cell_path=flow_cell_directory, bcl_converter=bcl_converter)
+
+
+def append_flow_cell_name_to_fastq_file_path(fastq_file_path: Path, flow_cell_name: str) -> Path:
```

Comment:
Append is a little misleading since that usually means attaching to the end, maybe prepend or add is less confusing

Reply:
fixed